### DISCUSSION TITLE
My Project got featured on Microsoft Blog

### DISCUSSION THREAD

**Discussion by Rishit-dagli** [EddieHubCommunity/support#1789] [Show and tell]
Our project got featured on Microsoft Blog 🥰  Check out how we started this out as a hackathon project and built it!

https://techcommunity.microsoft.com/t5/green-tech-blog/plant-ai-student-ambassador-green-a-thon-activity-report/ba-p/2348875

**Comment by ashwinexe** [comment]
That's an amazing project @Rishit-dagli I am inspired by you a lot  😄

**Reply by Rishit-dagli** [reply]
Thanks so much @ashwinexe !
